test/algorithms/multi_simplify.cpp

```diff
@@ -1,17 +1,12 @@
 // Boost.Geometry (aka GGL, Generic Geometry Library)
 //
-// Copyright (c) 2007-2012 Barend Gehrels, Amsterdam, the Netherlands.
+// Copyright (c) 2007-2015 Barend Gehrels, Amsterdam, the Netherlands.
 // Use, modification and distribution is subject to the Boost Software License,
 // Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
 // http://www.boost.org/LICENSE_1_0.txt)
 
 #include <geometry_test_common.hpp>
 
-#include <boost/geometry/multi/algorithms/simplify.hpp>
-
-#include <boost/geometry/io/wkt/wkt.hpp>
```

Comment:
It is included by `test_simplify.hpp` which is why I removed it.
Typically when we have a supporting `.hpp` file for a unit test, we do not the includes twice.
At least this is my understanding and experience with the unit tests I have seen so far.